app/controllers/webhook_controller.rb

```diff
@@ -10,6 +10,33 @@ def client
     }
   end
 
+  def search_shop(lat, lng)
```

Comment:
```suggestion
  def search_shop_names(lat, lng)
```

返り値がshop名の配列だと思うので、こういうメソッド名にした方が実態に近くなりますかね？